django_celery_beat/models.py

```diff
@@ -8,6 +8,7 @@
 
 import timezone_field
 from celery import current_app, schedules
+from celery.schedules import crontab
```

Comment:
```suggestion
```
Fix the failing pre-commit tests:
* https://results.pre-commit.ci/run/github/64884021/1712120266.lFMW0rs6QVShQA0QPquiuQ

Reply:
Thanks!